lib/Controller/Schedule.php

```diff
@@ -2554,6 +2554,11 @@ public function grid(Request $request, Response $response)
                         !empty($repeatsUntil) ? ' until ' . $repeatsUntil : ''
                     ))
                 );
+
+                $event->setUnmatchedProperty(
+                    'scheduleExclusions',
+                    $this->scheduleExclusionFactory->query(null, ['eventId' => $event->eventId])
```

Comment:
This will be too intensive to return without paging. We will need a different strategy to do it.